src/footer/_footer.scss

```diff
@@ -0,0 +1,174 @@
+@include govuk-exports("footer") {
+
+  // TODO: How do these colours fit into the colour scale?
+  $govuk-footer-background: $govuk-grey-3;
+  $govuk-footer-border: #a1acb2;
+  $govuk-footer-border-top: $govuk-footer-border;
+  $govuk-footer-text: #454a4c;
+  $govuk-footer-link: $govuk-footer-text;
+  $govuk-footer-link-hover: #171819;
+
+  .govuk-c-footer {
+    @include govuk-font-regular;
+    @include govuk-typography-responsive($govuk-font-16);
+
+    padding-top: $govuk-spacing-scale-7;
+    padding-bottom: $govuk-spacing-scale-5;
+
+    border-top: 1px solid $govuk-footer-border-top;
+    color: $govuk-footer-text;
+    background: $govuk-footer-background;
+  }
+
+  .govuk-c-footer__link {
+    @include govuk-focusable-fill;
+
+    &:link,
+    &:visited {
+      color: $govuk-footer-link;
+    }
+
+    &:hover,
+    &:active {
+      color: $govuk-footer-link-hover;
+    }
+  }
+
+  .govuk-c-footer__meta::after {
+    content: " ";
+    display: table;
+    clear: both;
+  }
+
+  .govuk-c-footer__meta {
+    display: flex;
+    margin-right: -($govuk-gutter / 2);
+    margin-left: -($govuk-gutter / 2);
+    flex-wrap: wrap;
+    align-items: flex-end;
+    justify-content: center;
+  }
+
+  .govuk-c-footer__meta-item {
+    margin-right: ($govuk-gutter / 2);
+    margin-bottom: $govuk-spacing-scale-5;
+    margin-left: ($govuk-gutter / 2);
+  }
+
+  .govuk-c-footer__meta-item--grow {
+    flex: 1;
+    @include mq ($until: tablet) {
+      flex-basis: 320px;
+    }
+  }
+
+  .govuk-c-footer__licence-logo {
+    display: inline-block;
+    margin-right: $govuk-spacing-scale-2;
+    @include mq ($until: desktop) {
+      margin-bottom: $govuk-spacing-scale-3;
+    }
+    vertical-align: top;
+  }
+
+  .govuk-c-footer__licence-description {
+    display: inline-block;
+  }
+
+  .govuk-c-footer__copyright-logo {
+    display: block;
+    min-width: 125px; // Based on the govuk-crest-2x.png image width: 250px/2
```

Comment:
Given that these numbers appear in a number of places with comments, it might be a good idea to break them out into variables:

```scss
.govuk-c-footer__copyright-logo {
  // Dimensions of the crest image at retina resolution
  $crest-image-width: 125px;
  $crest-image-height: 102px;

  display: block;
  min-width: $crest-image-width;
  // ...
}
```

Reply:
Good shout